usr/src/uts/armv8/Makefile.rules

```diff
@@ -38,6 +38,10 @@ $(OBJS_DIR)/%.o:		$(UTSBASE)/armv8/os/%.c
 	$(COMPILE.c) -I$(SRC)/contrib/libfdt -_gcc=-fno-stack-protector -o $@ $<
 	$(CTFCONVERT_O)
 
+$(OBJS_DIR)/%.o:		$(UTSBASE)/armv8/boot/%.c
+	$(COMPILE.c) -_gcc=-fno-stack-protector -o $@ $<
```

Comment:
Do we want to turn `-_gcc=-fno-stack-protector` into a `$(BOOT_CFLAGS)` or an `$(EARLY_CFLAGS)`?

Reply:
This is a good idea - I'll get that done.